tests/multinode.rs

```diff
@@ -1,38 +1,69 @@
 #[macro_use]
 extern crate log;
 extern crate bincode;
+extern crate serde_json;
 extern crate solana;
 
 use solana::bank::Bank;
 use solana::crdt::TestNode;
 use solana::crdt::{Crdt, ReplicatedData};
+use solana::entry::Entry;
 use solana::logger;
 use solana::mint::Mint;
 use solana::ncp::Ncp;
+use solana::payment_plan::PaymentPlan;
 use solana::server::Server;
 use solana::signature::{KeyPair, KeyPairUtil, PublicKey};
 use solana::streamer::default_window;
 use solana::thin_client::ThinClient;
+use solana::transaction::Instruction;
+use std::fs::File;
 use std::io;
 use std::io::sink;
+use std::io::BufRead;
+use std::io::BufReader;
 use std::net::UdpSocket;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{Arc, RwLock};
 use std::thread::sleep;
 use std::thread::JoinHandle;
 use std::time::Duration;
 
-fn validator(
+fn poll_file_for_entries(path: &str, num: usize) -> Option<Vec<Entry>> {
+    for _ in 0..10 {
+        let entries: Vec<Entry> = BufReader::new(File::open(path).unwrap())
+            .lines()
+            .filter_map(|line| {
+                if let Ok(entry) = serde_json::from_str(&line.unwrap()) {
+                    Some(entry)
+                } else {
+                    None
+                }
+            })
+            .collect();
+        if entries.len() == num {
+            return Some(entries);
+        }
+    }
+    None
+}
+
+fn bank_from_file(path: &str) -> (Bank, u64) {
+    let bank = Bank::default();
+    bank.process_ledger(&path)
```

Comment:
That compiles!?

Reply:
`cargo watch -x build` skips the tests :(